**Windows** [Build 4484](https://ci.wildfly.org/viewLog.html?buildId=14549&buildTypeId=WF_PullRequest_Windows) outcome was **FAILURE** using a merge of 3a6297cdc07728b378fda3f3bdd828fa600829dc
Summary: Tests failed: 40 (36 new), passed: 3290, ignored: 195 Build time: 01:11:01
### Failed tests
```
org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithSameRuntimeNameOnSameServerGroup: java.lang.AssertionError: Failed to execute line 'deploy --server-groups=main-server-group C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\HelloServlet.war --runtime-name=SimpleServlet.war --name=simple1': org.jboss.as.cli.CommandFormatException: {"WFLYDC0074: Operation failed or was rolled back on all servers. Server failures:" => {"server-group" => {"main-server-group" => {"host" => {"master" => {"main-one" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0063: Composite operation was rolled back"}}},"slave" => {"main-three" => "WFLYHC0153: Channel closed"}}}}}}
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithSameRuntimeNameOnSameServerGroup(DomainDeployWithRuntimeNameTestCase.java:99)
------- Stdout: -------
[Host Controller] 16:57:50,778 INFO  [org.jboss.as.host.controller] (management-handler-thread - 3) WFLYHC0024: Stopping server test-one
[Server:test-one] 16:57:50,782 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 60) WFLYSRV0211: Suspending server with 0ms timeout.
16:57:50,784 INFO  [org.jboss.as.process.Server:test-one.status] (ProcessController-threads - 3) WFLYPC0019: Stopping process 'Server:test-one'
[Server:test-one] 16:57:50,785 INFO  [org.jboss.as.server] (Thread-2) WFLYSRV0220: Server shutdown has been requested.
[Server:test-one] 16:57:50,802 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-3) WFLYJCA0010: Unbound data source [java:jboss/datasources/ExampleDS]
[Server:test-one] 16:57:50,804 INFO  [org.wildfly.extension.messaging-activemq] (MSC service thread 1-7) WFLYMSGAMQ0006: Unbound messaging object to jndi name java:jboss/DefaultJMSConnectionFactory
[Server:test-one] 16:57:50,806 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 60) WFLYUT0022: Unregistered web context: /RolloutPlanTestCase
[Server:test-one] 16:57:50,806 INFO  [org.wildfly.extension.messaging-activemq] (ServerService Thread Pool -- 62) WFLYMSGAMQ0006: Unbound messaging object to jndi name java:jboss/exported/jms/RemoteConnectionFactory
[Server:test-one] 16:57:50,809 INFO  [org.jboss.as.connector.deployment] (MSC service thread 1-3) WFLYJCA0011: Unbound JCA ConnectionFactory [java:/JmsXA]
[Server:test-one] 16:57:50,819 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-2) WFLYJCA0019: Stopped Driver service with driver-name = h2
[Server:test-one] 16:57:50,819 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0019: Host default-host stopping
[Server:test-one] 16:57:50,825 INFO  [org.jboss.modcluster] (ServerService Thread Pool -- 60) MODCLUSTER000002: Initiating mod_cluster shutdown
[Server:test-one] 16:57:50,835 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment RolloutPlanTestCase.war (runtime-name: RolloutPlanTestCase.war) in 37ms
[Server:test-one] 16:57:50,836 INFO  [org.apache.activemq.artemis.ra] (ServerService Thread Pool -- 64) AMQ151003: resource adaptor stopped
[Server:test-one] 16:57:50,837 INFO  [com.arjuna.ats.jbossatx] (MSC service thread 1-7) ARJUNA032018: Destroying TransactionManagerService
[Server:test-one] 16:57:50,838 INFO  [com.arjuna.ats.jbossatx] (MSC service thread 1-7) ARJUNA032014: Stopping transaction recovery manager
[Server:test-one] 16:57:50,851 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 64) AMQ221002: Apache ActiveMQ Artemis Message Broker version 1.1.0.wildfly-016 [be3f88d6-081c-11e6-aa08-cf82a32b6522] stopped
[Server:test-one] 16:57:50,852 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-1) WFLYUT0008: Undertow HTTP listener default suspending
[Server:test-one] 16:57:50,853 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-1) WFLYUT0007: Undertow HTTP listener default stopped, was bound to [::1]:8080
[Server:test-one] 16:57:50,853 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0004: Undertow 1.3.19.Final stopping
[Host Controller] 16:57:50,860 INFO  [org.jboss.as.host.controller] (management I/O-2) WFLYHC0027: Unregistering server test-one
[Server:test-one] 16:57:50,867 INFO  [org.jboss.as] (MSC service thread 1-5) WFLYSRV0050: WildFly Full 10.1.0.Final-SNAPSHOT (WildFly Core 2.1.0.Final) stopped in 64ms
16:57:50,927 INFO  [org.jboss.as.process.Server:test-one.status] (reaper for Server:test-one) WFLYPC0011: Process 'Server:test-one' finished with an exit status of 0
[Host Controller] 16:57:50,928 INFO  [org.jboss.as.host.controller] (ProcessControllerConnection-thread - 2) WFLYHC0027: Unregistering server test-one
ERROR [org.jboss.as.cli.CommandContext] {
    "outcome" => "success",
    "result" => "STOPPED"
}
ERROR [org.jboss.as.cli.CommandContext] {
    "outcome" => "success",
    "result" => "STOPPED"
}
[Host Controller] 16:57:50,934 INFO  [org.jboss.as.host.controller] (management-handler-thread - 3) WFLYHC0024: Stopping server main-two
ERROR [org.jboss.as.cli.CommandContext] {
    "outcome" => "success",
    "result" => "STOPPED"
}
ERROR [org.jboss.as.cli.CommandContext] {
    "outcome" => "success",
    "result" => "DISABLED"
}
INFO  [org.jboss.as.cli.CommandContext] Warning! The CLI is running in a non-modular environment and cannot load commands from management extensions.
[Host Controller] 16:57:51,045 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\domains\CLITestSuite\master\data\content\cc\7cb34126fca69179d12c84922b593d389c1b00\content
[Server:main-one] 16:57:51,165 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "simple1" (runtime-name: "SimpleServlet.war")
[Server:main-one] 16:57:51,232 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 63) WFLYUT0021: Registered web context: /SimpleServlet
[Server:main-one] 16:57:51,236 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 27) WFLYSRV0022: Deploy of deployment "SimpleServlet.war" was rolled back with no failure message


org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithSameRuntimeNameOnDifferentServerGroup: java.lang.AssertionError: Failed to execute line 'deploy --server-groups=main-server-group C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\HelloServlet.war --runtime-name=SimpleServlet.war --name=simple1': org.jboss.as.cli.CommandFormatException: {"WFLYDC0074: Operation failed or was rolled back on all servers. Server failures:" => {"server-group" => {"main-server-group" => {"host" => {"master" => {"main-one" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0063: Composite operation was rolled back"}}},"slave" => {"main-three" => "WFLYHC0153: Channel closed"}}}}}}
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithSameRuntimeNameOnDifferentServerGroup(DomainDeployWithRuntimeNameTestCase.java:114)
------- Stdout: -------
[Server:main-one] 16:57:51,310 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-7) WFLYSRV0027: Starting deployment of "simple1" (runtime-name: "SimpleServlet.war")
[Server:main-one] 16:57:51,347 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 62) WFLYUT0021: Registered web context: /SimpleServlet
[Server:main-one] 16:57:51,352 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 27) WFLYSRV0022: Deploy of deployment "SimpleServlet.war" was rolled back with no failure message
[Server:main-one] 16:57:51,352 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 62) WFLYUT0022: Unregistered web context: /SimpleServlet
[Server:main-one] 16:57:51,358 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0028: Stopped deployment simple1 (runtime-name: SimpleServlet.war) in 6ms
ERROR [org.jboss.as.cli.CommandContext] Undeploy failed: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0216: Management resource '[(\"deployment\" => \"simple1\")]' not found"}}
ERROR [org.jboss.as.cli.CommandContext] Undeploy failed: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0216: Management resource '[(\"deployment\" => \"simple2\")]' not found"}}


org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithDifferentRuntimeNameOnDifferentServerGroup: java.lang.AssertionError: Failed to execute line 'deploy --server-groups=main-server-group C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\HelloServlet.war --runtime-name=SimpleServlet.war --name=simple1': org.jboss.as.cli.CommandFormatException: {"WFLYDC0074: Operation failed or was rolled back on all servers. Server failures:" => {"server-group" => {"main-server-group" => {"host" => {"master" => {"main-one" => {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0063: Composite operation was rolled back"}}},"slave" => {"main-three" => "WFLYHC0153: Channel closed"}}}}}}
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DomainDeployWithRuntimeNameTestCase.testDeployWithDifferentRuntimeNameOnDifferentServerGroup(DomainDeployWithRuntimeNameTestCase.java:130)
------- Stdout: -------
[Server:main-one] 16:57:51,436 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-3) WFLYSRV0027: Starting deployment of "simple1" (runtime-name: "SimpleServlet.war")
[Server:main-one] 16:57:51,480 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 63) WFLYUT0021: Registered web context: /SimpleServlet
[Server:main-one] 16:57:51,488 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 27) WFLYSRV0022: Deploy of deployment "SimpleServlet.war" was rolled back with no failure message
[Server:main-one] 16:57:51,488 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 63) WFLYUT0022: Unregistered web context: /SimpleServlet
[Server:main-one] 16:57:51,501 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-1) WFLYSRV0028: Stopped deployment simple1 (runtime-name: SimpleServlet.war) in 12ms
ERROR [org.jboss.as.cli.CommandContext] Undeploy failed: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0216: Management resource '[(\"deployment\" => \"simple1\")]' not found"}}
ERROR [org.jboss.as.cli.CommandContext] Undeploy failed: {"WFLYCTL0062: Composite operation failed and was rolled back. Steps that failed:" => {"Operation step-1" => "WFLYCTL0216: Management resource '[(\"deployment\" => \"simple2\")]' not found"}}


org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testXaDataSource: java.lang.AssertionError: Failed to execute line 'xa-data-source add --profile=default --jndi-name=java:jboss/datasources/TestXADS --name=java:jboss/datasources/TestXADS --driver-name=h2 --xa-datasource-properties=ServerName=localhost,PortNumber=50011': org.jboss.as.cli.CommandLineException: {"WFLYDC0074: Operation failed or was rolled back on all servers. Server failures:" => {"server-group" => {"main-server-group" => {"host" => {"slave" => {"main-three" => "WFLYHC0153: Channel closed"}}}}}}
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testAddXaDataSource(DataSourceTestCase.java:158)
	at org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testXaDataSource(DataSourceTestCase.java:83)
------- Stdout: -------
INFO  [org.jboss.as.cli.CommandContext] Warning! The CLI is running in a non-modular environment and cannot load commands from management extensions.
[Server:main-one] 16:57:51,762 WARN  [org.jboss.as.connector.subsystems.datasources.AbstractDataSourceService$AS7DataSourceDeployer] (MSC service thread 1-8) IJ020016: Missing <recovery> element. XA recovery disabled for: java:jboss/datasources/TestXADS
[Server:main-one] 16:57:51,763 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-5) WFLYJCA0001: Bound data source [java:jboss/datasources/TestXADS]
[Server:main-one] 16:57:51,772 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-6) WFLYJCA0010: Unbound data source [java:jboss/datasources/TestXADS]


org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testDataSource: java.lang.AssertionError: Failed to execute line 'data-source add --profile=default --jndi-name=java:jboss/datasources/TestDS_h2 --name=java:jboss/datasources/TestDS_h2 --driver-name=h2 --connection-url=jdbc:h2:mem:test;DB_CLOSE_DELAY=-1': org.jboss.as.cli.CommandLineException: {"WFLYDC0074: Operation failed or was rolled back on all servers. Server failures:" => {"server-group" => {"main-server-group" => {"host" => {"slave" => {"main-three" => "WFLYHC0153: Channel closed"}}}}}}
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testAddDataSource(DataSourceTestCase.java:109)
	at org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testDataSource(DataSourceTestCase.java:76)
------- Stdout: -------
[Server:main-one] 16:57:51,875 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-4) WFLYJCA0001: Bound data source [java:jboss/datasources/TestDS_h2]
[Server:main-one] 16:57:51,885 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-4) WFLYJCA0010: Unbound data source [java:jboss/datasources/TestDS_h2]


org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testDataSourcewithHotDeployedJar: java.lang.AssertionError: Failed to execute line 'deploy --all-server-groups C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\foodriver.jar': org.jboss.as.cli.CommandFormatException: WFLYDC0022: Operation failed or was rolled back on all servers.
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:165)
	at org.jboss.as.test.integration.management.util.CLIWrapper.sendLine(CLIWrapper.java:187)
	at org.jboss.as.test.integration.domain.management.cli.DataSourceTestCase.testDataSourcewithHotDeployedJar(DataSourceTestCase.java:90)
------- Stdout: -------
[Host Controller] 16:57:51,911 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\domains\CLITestSuite\master\data\content\bb\d58ea8401215982e8f90ba650368d5e384c016\content
[Server:main-one] 16:57:51,963 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-1) WFLYSRV0027: Starting deployment of "foodriver.jar" (runtime-name: "foodriver.jar")
[Server:main-one] 16:57:51,974 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0005: Deploying non-JDBC-compliant driver class org.jboss.as.test.integration.domain.driver.FooDriver (version 0.0)
[Server:main-one] 16:57:51,984 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0018: Started Driver service with driver-name = foodriver.jar
[Server:other-two] 16:57:51,990 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0027: Starting deployment of "foodriver.jar" (runtime-name: "foodriver.jar")
[Server:other-two] 16:57:52,004 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-6) WFLYJCA0005: Deploying non-JDBC-compliant driver class org.jboss.as.test.integration.domain.driver.FooDriver (version 0.0)
[Server:other-two] 16:57:52,006 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-6) WFLYJCA0018: Started Driver service with driver-name = foodriver.jar
[Server:other-two] 16:57:52,050 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 60) WFLYSRV0022: Deploy of deployment "foodriver.jar" was rolled back with no failure message
[Server:other-two] 16:57:52,058 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0019: Stopped Driver service with driver-name = foodriver.jar
[Server:other-two] 16:57:52,066 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0028: Stopped deployment foodriver.jar (runtime-name: foodriver.jar) in 9ms
[Server:main-one] 16:57:52,067 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 27) WFLYSRV0022: Deploy of deployment "foodriver.jar" was rolled back with no failure message
[Server:main-one] 16:57:52,068 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0019: Stopped Driver service with driver-name = foodriver.jar
[Server:main-one] 16:57:52,075 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment foodriver.jar (runtime-name: foodriver.jar) in 6ms


org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testFullReplaceDifferentFile: java.lang.AssertionError: "WFLYDC0022: Operation failed or was rolled back on all servers."
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:220)
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:212)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.executeOnMaster(DeploymentManagementTestCase.java:819)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testDeploymentViaStream(DeploymentManagementTestCase.java:248)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testFullReplaceDifferentFile(DeploymentManagementTestCase.java:577)
------- Stdout: -------
[Host Controller] 16:58:17,293 INFO  [org.jboss.as.repository] (management-handler-thread - 4) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\domains\DomainTestSuite\master\data\content\75\5f0058b46a9ae6a3cc6309c77837b4b6c9f78d\content
[Server:main-one] 16:58:17,582 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:other-two] 16:58:18,211 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:18,233 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-3) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:18,265 ERROR [org.jboss.as.controller.management-operation] (ServerService Thread Pool -- 26) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "test.war")]) - failure description: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:18,267 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 26) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
[Server:main-three] {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:18,317 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 37ms
[Server:main-three] 16:58:18,318 INFO  [org.jboss.as.controller] (ServerService Thread Pool -- 26) WFLYCTL0183: Service status report
[Server:main-three] WFLYCTL0186:   Services which failed to start:      service jboss.undertow.listener.default: org.jboss.msc.service.StartException in service jboss.undertow.listener.default: Could not start http listener
[Server:main-three] 
[Server:main-one] 16:58:18,666 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0021: Registered web context: /test
[Server:other-two] 16:58:18,768 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 14) WFLYUT0021: Registered web context: /test
[Server:other-two] 16:58:18,803 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:18,793 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:18,810 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:18,814 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 14) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:18,846 INFO  [org.hibernate.validator.internal.util.Version] (MSC service thread 1-6) HV000001: Hibernate Validator 5.2.4.Final
[Server:main-one] 16:58:18,860 INFO  [org.hibernate.validator.internal.util.Version] (MSC service thread 1-5) HV000001: Hibernate Validator 5.2.4.Final
[Server:other-two] 16:58:18,976 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 172ms
[Server:main-one] 16:58:18,994 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 201ms
Failed response:
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "rolled-back" => true


org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedExplodedFullReplaceManaged: java.lang.AssertionError: "WFLYDC0022: Operation failed or was rolled back on all servers."
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:220)
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:212)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.executeOnMaster(DeploymentManagementTestCase.java:819)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testDeploymentViaStream(DeploymentManagementTestCase.java:248)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedExplodedFullReplaceManaged(DeploymentManagementTestCase.java:651)
------- Stdout: -------
[Host Controller] 16:58:19,025 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\domain\target\domains\DomainTestSuite\master\data\content\ad\591c46622ac4dbb9f898cc80e5048d227bf671\content
[Server:main-one] 16:58:19,111 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:19,172 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:other-two] 16:58:19,170 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:19,212 ERROR [org.jboss.as.controller.management-operation] (ServerService Thread Pool -- 26) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "test.war")]) - failure description: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:19,213 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 26) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
[Server:main-three] {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:19,251 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 36ms
[Server:main-three] 16:58:19,265 INFO  [org.jboss.as.controller] (ServerService Thread Pool -- 26) WFLYCTL0183: Service status report
[Server:main-three] WFLYCTL0186:   Services which failed to start:      service jboss.undertow.listener.default: org.jboss.msc.service.StartException in service jboss.undertow.listener.default: Could not start http listener
[Server:main-three] 
[Server:main-one] 16:58:19,343 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0021: Registered web context: /test
[Server:other-two] 16:58:19,415 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0021: Registered web context: /test
[Server:main-one] 16:58:19,430 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:19,432 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:19,440 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:19,438 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 7ms
[Server:other-two] 16:58:19,445 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:19,455 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 11ms
Failed response:
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-3" => {


org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testManagedReplaceUnmanaged: java.lang.AssertionError: "WFLYDC0022: Operation failed or was rolled back on all servers."
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:220)
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:212)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.executeOnMaster(DeploymentManagementTestCase.java:815)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedArchiveDeployment(DeploymentManagementTestCase.java:310)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testManagedReplaceUnmanaged(DeploymentManagementTestCase.java:493)
------- Stdout: -------
[Server:main-one] 16:58:19,531 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:19,567 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:19,575 ERROR [org.jboss.as.controller.management-operation] (ServerService Thread Pool -- 26) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "test.war")]) - failure description: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:19,579 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 26) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
[Server:main-three] {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:other-two] 16:58:19,584 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:other-two] 16:58:19,629 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0021: Registered web context: /test
[Server:main-three] 16:58:19,619 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 33ms
[Server:main-three] 16:58:19,630 INFO  [org.jboss.as.controller] (ServerService Thread Pool -- 26) WFLYCTL0183: Service status report
[Server:main-three] WFLYCTL0186:   Services which failed to start:      service jboss.undertow.listener.default: org.jboss.msc.service.StartException in service jboss.undertow.listener.default: Could not start http listener
[Server:main-three] 
[Server:main-one] 16:58:19,644 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0021: Registered web context: /test
[Server:other-two] 16:58:19,660 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:19,659 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:other-two] 16:58:19,662 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0022: Unregistered web context: /test
[Server:main-one] 16:58:19,663 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:19,676 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 16ms
[Server:main-one] 16:58:19,678 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 19ms
Failed response:
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-3" => {
            "outcome" => "failed",


org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedArchiveReplaceManaged: java.lang.AssertionError: "WFLYDC0022: Operation failed or was rolled back on all servers."
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:220)
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:212)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.executeOnMaster(DeploymentManagementTestCase.java:819)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testDeploymentViaStream(DeploymentManagementTestCase.java:248)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedArchiveReplaceManaged(DeploymentManagementTestCase.java:457)
------- Stdout: -------
[Server:main-one] 16:58:20,017 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-one] 16:58:20,064 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0021: Registered web context: /test
[Server:main-three] 16:58:20,075 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:20,097 ERROR [org.jboss.as.controller.management-operation] (ServerService Thread Pool -- 26) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "test.war")]) - failure description: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:20,099 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 26) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
[Server:main-three] {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:20,115 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 2ms
[Server:main-three] 16:58:20,116 INFO  [org.jboss.as.controller] (ServerService Thread Pool -- 26) WFLYCTL0183: Service status report
[Server:main-three] WFLYCTL0186:   Services which failed to start:      service jboss.undertow.listener.default: org.jboss.msc.service.StartException in service jboss.undertow.listener.default: Could not start http listener
[Server:main-three] 
[Server:other-two] 16:58:20,198 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-5) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:other-two] 16:58:20,238 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0021: Registered web context: /test
[Server:main-one] 16:58:20,246 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:main-one] 16:58:20,249 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:20,251 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:other-two] 16:58:20,254 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0022: Unregistered web context: /test
[Server:other-two] 16:58:20,267 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-3) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 15ms
[Server:main-one] 16:58:20,270 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 22ms
Failed response:
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-3" => {
            "outcome" => "failed",


org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedArchiveDeployment: java.lang.AssertionError: "WFLYDC0022: Operation failed or was rolled back on all servers."
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:220)
	at org.jboss.as.test.integration.domain.management.util.DomainTestSupport.validateResponse(DomainTestSupport.java:212)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.executeOnMaster(DeploymentManagementTestCase.java:815)
	at org.jboss.as.test.integration.domain.suites.DeploymentManagementTestCase.testUnmanagedArchiveDeployment(DeploymentManagementTestCase.java:310)
------- Stdout: -------
[Server:main-one] 16:58:20,746 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-2) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:other-two] 16:58:20,765 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:20,802 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0027: Starting deployment of "test.war" (runtime-name: "test.war")
[Server:main-three] 16:58:20,872 ERROR [org.jboss.as.controller.management-operation] (ServerService Thread Pool -- 26) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "test.war")]) - failure description: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-three] 16:58:20,873 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 26) WFLYSRV0021: Deploy of deployment "test.war" was rolled back with the following failure message: 
[Server:main-three] {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
[Server:main-three]     "Services that were unable to start:" => ["jboss.deployment.unit.\"test.war\".PARSE"],
[Server:main-three]     "Services that may be the cause:" => [
[Server:main-three]         "jboss.http-upgrade-registry.default",
[Server:main-three]         "jboss.remoting.remotingConnectorInfoService.http-remoting-connector"
[Server:main-three]     ]
[Server:main-three] }}
[Server:main-one] 16:58:20,884 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0021: Registered web context: /test
[Server:main-three] 16:58:20,878 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 5ms
[Server:main-three] 16:58:20,879 INFO  [org.jboss.as.controller] (ServerService Thread Pool -- 26) WFLYCTL0183: Service status report
[Server:main-three] WFLYCTL0186:   Services which failed to start:      service jboss.undertow.listener.default: org.jboss.msc.service.StartException in service jboss.undertow.listener.default: Could not start http listener
[Server:main-three] 
[Server:other-two] 16:58:20,891 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0021: Registered web context: /test
[Server:main-one] 16:58:20,899 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:other-two] 16:58:20,900 ERROR [org.jboss.as.server] (ServerService Thread Pool -- 9) WFLYSRV0022: Deploy of deployment "test.war" was rolled back with no failure message
[Server:other-two] 16:58:20,901 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 15) WFLYUT0022: Unregistered web context: /test
[Server:main-one] 16:58:20,901 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 10) WFLYUT0022: Unregistered web context: /test
[Server:main-one] 16:58:20,909 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 9ms
[Server:other-two] 16:58:20,913 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-1) WFLYSRV0028: Stopped deployment test.war (runtime-name: test.war) in 12ms
Failed response:
{
    "outcome" => "failed",
    "result" => {
        "step-1" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-2" => {
            "outcome" => "failed",
            "rolled-back" => true
        },
        "step-3" => {
            "outcome" => "failed",
            "rolled-back" => true



##### there are 30 more failed tests, see build details
```

--- a/clustering/singleton/api/src/main/java/org/wildfly/clustering/singleton/SingletonServiceBuilderFactory.java
+++ b/clustering/singleton/api/src/main/java/org/wildfly/clustering/singleton/SingletonServiceBuilderFactory.java
@@ -29,10 +29,6 @@ import org.jboss.msc.service.ServiceName;
  * @author Paul Ferraro
  */
 public interface SingletonServiceBuilderFactory extends SingletonPolicy {
-    /** Use {@link org.wildfly.clustering.singleton.spi.SingletonServiceName} to generate an appropriate {@link ServiceName}.
-    @Deprecated
-    ServiceName SERVICE_NAME = ServiceName.JBOSS.append("clustering", "singleton", "builder");
-
     /**
      * Creates a singleton service builder.
      * @param name the name of the service

